README.md

```diff
@@ -1,55 +1,55 @@
 # Dash Bio
 
-[![CircleCI](https://circleci.com/gh/plotly/dash-bio/tree/master.svg?style=svg)](https://circleci.com/gh/plotly/dash-bio)
```

Comment:
When did we decide to remove the CI status badge?

Reply:
We may want to move it right next to the level-1 "Dash Bio" title, it's quite conventional, or further down in the README, if you prefer.

Reply:
Resolved in 1babfd7.

Reply:
Ok, but the line added in 1babfd7 is prefixed with two whitespaces.

Reply:
Fixed in ea4567a.